packages/i3.rb

```diff
@@ -40,6 +27,20 @@ def self.install
     system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
     Dir.chdir "#{CREW_DEST_PREFIX}/bin" do
       system "for f in \$(ls #{CREW_BUILD}-*); do g=\$(echo \$f | sed 's,#{CREW_BUILD}-,,'); ln -sf \$f \$g; done"
+     system "echo '#!/bin/sh' > starti3"
+     system "echo 'stopsommelier' >> starti3"
+     system "echo 'export DISPLAY=100.115.92.2:0' >> starti3"
+     system "echo 'i3 \"$@\"' >> starti3"
+     system "echo 'export DISPLAY=:0' >> starti3"
+     system "echo 'initsommelier' >> starti3"
```

Comment:
`system "echo 'startsommelier'` >> starti3"` will likely work better here.

Reply:
Might want to check if sommelier is installed first in the script since this package does not include sommelier as a dependency.  Otherwise, the sommelier commands will fail ungracefully.

Reply:
`sommelier` isn't already in the dependencies?

Reply:
> `system "echo 'startsommelier' >> starti3"` will likely work better here.

Done